src/atoms/Button/Button.stories.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
```

Comment:
Have changed the stories accidentally, forgot that native and web is sharing stories.

Took a look on the original stories, and only one story demostrating `align` property of the `Button` looks missing to me. I can add that but would like to know if I should instead revert the stories to the original sense or like this?